openbas-api/src/main/java/io/openbas/telemetry/PyroscopeConfig.java

```diff
@@ -0,0 +1,88 @@
+package io.openbas.telemetry;
+
+import io.pyroscope.javaagent.PyroscopeAgent;
+import io.pyroscope.labels.LabelsSet;
+import io.pyroscope.labels.Pyroscope;
+import io.swagger.v3.oas.annotations.Operation;
+import java.lang.reflect.Method;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.aspectj.lang.annotation.Pointcut;
+import org.aspectj.lang.reflect.MethodSignature;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.*;
+
+@Aspect
+@Component
```

Comment:
You can use @ConditionalOnProperty(name = "", havingValue = "", matchIfMissing = true)